dts/bindings/ethernet/atmel,sam-gmac.yaml

```diff
@@ -0,0 +1,17 @@
+# Copyright (c) 2020 Stephanos Ioannidis <root@stephanos.io>
+# SPDX-License-Identifier: Apache-2.0
+
+description: Atmel SAM-family GMAC Ethernet
+
+compatible: "atmel,sam-gmac"
+
+include: ethernet.yaml
+
+properties:
+    reg:
+      required: true
+
+    queue-num:
+      type: int
+      required: true
+      description: Number of hardware TX and RX queues
```

Comment:
Just to be clear is the number of TX + RX queues?  (I'm assuming that there are the same number of RX & TX queues)

Reply:
Yes, this is the total number of hardware queues available (including TX and RX).